src/Nelmio/Alice/Fixtures/Loader.php

```diff
@@ -245,8 +250,8 @@ public function addPopulator(Populator\Methods\MethodInterface $populator)
     /**
      * parses a file at the given filename
      *
-     * @param string filename
-     * @return array data
+     * @param  string $filename
+     * @return array  data
      */
     protected function parseFile($filename)
```

Comment:
You really can't - it's different for any dev's implementation, since they can register custom parsers.